### DISCUSSION TITLE
Help setting this up. Currently followed Trashguides for all other -arr programs a few years ago.

### DISCUSSION THREAD

**Discussion by jordanmontano** [RandomNinjaAtk/arr-scripts#195] [Q&A] [answered]
Hello!

So basically I was wondering where I should put the script, I have Lidar already running in a docker container (on Synology.) I would love to add this but I think I am placing it in the wrong area and would love any help with setup.

**Comment by RandomNinjaAtk** [comment, marked as answer]
https://github.com/RandomNinjaAtk/arr-scripts/tree/main/lidarr#installationsetup

**Reply by jordanmontano** [reply]
Thanks, I saw this but just wanted to make sure I was understanding correctly. Do I just need to nest both of those folders: 

/custom-cont-init.d
/custom-services.d

inside my docker/appdata/lidarr folder?

**Reply by RandomNinjaAtk** [reply]
That should be fine.
